esy-solve/Resolver.re

```diff
@@ -498,7 +506,7 @@ let package =
           resolver.npmRegistry,
           (),
         );
-
+      let pkg = patch_overrides(pkg);
```

Comment:
This is clearly not ideal, but the problem is that both npm and opam have cached manifests so using patching them on the source would change everything.